### PR TITLE
Add jupyterlab-kernelspy

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bollwyvl** [comment]
can help maintain, lmk if you run into any issues...

**Comment by jtpio** [comment]
Waiting for this one to be fixed upstream: https://github.com/jupyterlab-contrib/jupyterlab-kernelspy/pull/34

**Comment by jtpio** [comment]
Thanks @bollwyvl, I've added you to the list of maintainers.

File: recipes/jupyterlab-kernelspy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{% set version = "3.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 557ff653901ef3c236fbfc549e55910ba3ffc9f63d7fbef51c75fa58772f2e4c
```
Comment (bollwyvl) [inline recipes/jupyterlab-kernelspy/meta.yaml:11]: if we want it now:
```yaml
source:
  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
    sha256: 557ff653901ef3c236fbfc549e55910ba3ffc9f63d7fbef51c75fa58772f2e4c
  - url: https://raw.githubusercontent.com/jupyterlab-contrib/jupyterlab-kernelspy/{{ version }}/LICENSE.md
    sha256: abc123 # ...but 3.0.2 isn't tagged
```

File: recipes/jupyterlab-kernelspy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  summary: A JupyterLab extension for inspecting messages to/from a kernel
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - fcollonval
+    - jtpio
+    - vidartf
```
Comment (jtpio) [inline recipes/jupyterlab-kernelspy/meta.yaml:46]: cc @fcollonval @vidartf if you would like to be listed here

Comment (fcollonval) [reply recipes/jupyterlab-kernelspy/meta.yaml:46]: Thanks @jtpio I agree to be maintainer on the conda recipe too.

**Comment by jtpio** [comment]
All green now.

**Comment by jtpio** [comment]
cc @conda-forge/help-python, this is ready for review.

Thanks!

**Comment by bollwyvl** [comment]
Also cc @conda-forge/staged-recipes

**Review by dopplershift — APPROVED** [review]

**Pull request merged**